package.json

```diff
@@ -5,11 +5,33 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "canned -p 5000 ./server"
+    "start": "node ./app.js",
+    "startnew": "json-server -p 5000 ./server/db.json",
+    "gulp": "gulp"
   },
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "canned": "^0.3.12"
+    "bootstrap": "^3.4.1",
```

Comment:
can we have the project without this module?

Reply:
yeah... uninstalled...